tools/highlighter-helper/src/client/composables/collapsedLines.ts

```diff
@@ -0,0 +1,14 @@
+export const setupCollapsedLines = ({
+  selector = 'div[class*="language-"].has-collapsed-lines > .collapsed-lines',
+}: { selector?: string } = {}): void => {
+  if (__VUEPRESS_SSR__) return
+  window.addEventListener('click', (e) => {
```

Comment:
I would prefer `useEventListener`, I discourage to see `__VUEPRESS_SSR__` as a judgement to visit window outside `onMounted`